app/services/ScratchService.scala

```diff
@@ -26,16 +26,20 @@ import repositories.ScratchRepository
 import play.api.Logger
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
+import config.AppConfig
 
 @Singleton
-class ScratchService @Inject() (repository: ScratchRepository, pageBuilder: PageBuilder, securedProcessBuilder: SecuredProcessBuilder) {
+class ScratchService @Inject() (repository: ScratchRepository,
+                                pageBuilder: PageBuilder,
+                                securedProcessBuilder: SecuredProcessBuilder,
+                                implicit val c: AppConfig) {
   val logger = Logger(getClass)
 
   def save(process: JsObject): Future[RequestOutcome[UUID]] =
     guidancePages(pageBuilder, securedProcessBuilder, process).fold(
       err => Future.successful(Left(err)),
-      _ =>
-        repository.save(process).map {
+      result =>
+        repository.save(result._3).map {
```

Comment:
Not a problem, just a comment.  I know using a tuple simplifies multiple returns, but it makes the code a bit less intuitive as you have to view the method called to see what entry 3 actually is.

Reply:
will update